repo/mapping.go

```diff
@@ -69,34 +69,57 @@ func (d *DeviceMapping) GetImageTitles() []string {
 	return r
 }
 
-// Dir returns directory of a local repo `.iotit/images/{name}`
+// Dir - returns directory of a local repo `.iotit/images/{name}`
 func (d *DeviceMapping) Dir() string {
 	return filepath.Join(ImageDir, d.dir)
 }
 
+func (d *DeviceMapping) FindImage(image string) error {
```

Comment:
__[golint]__
* [golint] exported method DeviceMapping.FindImage should have comment or be unexported

<!-- Comment by [SideCI](https://sideci.com) -->